cfgov/v1/atomic_elements/organisms.py

```diff
@@ -165,7 +165,7 @@ class InfoUnitGroup2575Only(InfoUnitGroup):
     )
 
     class Meta:
-        label = 'Info unit group'
+        label = 'IUG 25/75 (deprecated)'
```

Comment:
Can we add a `TODO` comment here that describes what state we have to reach to be able to remove this bit?